### PR TITLE
IGAPP-xx: Fix node version in readme

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ to make these changes available in both projects.
 
 - Rate our Integreat app in the [PlayStore](https://play.google.com/store/apps/details?id=tuerantuer.app.integreat)
   and the [Apple App Store](https://apps.apple.com/ae/app/integreat/id1072353915).
-- Install [nodejs](https://nodejs.org/). At least v14.17 is required.
+- Install [nodejs](https://nodejs.org/). (Version >= 15.0 and <= 16.13.0 required)
```
Comment (klinzo) [inline README.md:54]: Why is it restricted not to be higher than `16.13.0`?

**Review by klinzo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SteffiStuffel) [reply README.md:54]: Otherwise this error will occur: `TypeError: Cannot read properties of undefined (reading 'transformFile')`

Comment (klinzo) [reply README.md:54]: Did you find an issue for this?

Comment (Taggotty) [reply README.md:54]: I think setting `export NODE_OPTIONS=--openssl-legacy-provider` fixes this issue https://github.com/nodejs/node/issues/40455

**Review by Taggotty — APPROVED** [review]

**Pull request merged**